src/app/views/query-response/adaptive-cards/AdaptiveCard.tsx

```diff
@@ -99,4 +100,6 @@ function mapDispatchToProps(dispatch: Dispatch): object {
 
 // @ts-ignore
 const IntlAdaptiveCard = injectIntl(AdaptiveCard);
-export default connect(mapStateToProps, mapDispatchToProps)(IntlAdaptiveCard);
+// @ts-ignore
+const trackedComponent = telemetry.trackReactComponent(IntlAdaptiveCard, AdaptiveCard.name);
+export default connect(mapStateToProps, mapDispatchToProps)(trackedComponent);
```

Comment:
From the data we've collected so far in test, the only property being auto-captured is `Component Name`. The other properties are those I've added in the filter function, `IsAuthenticated` and `ApplicationName`.

Reply:
According to the code in withAITracking, it captures all properties. If we ever add another property that is not intended to be captured, it will be captured. Maybe I'm reading this incorrectly

Reply:
This piece of code from [here](https://github.com/microsoft/ApplicationInsights-JS/blob/5bd00322e30483552d75a09df81094d29237a967/extensions/applicationinsights-react-js/src/withAITracking.tsx#L69),  tells me that `Component Name`  is the only property captured. 
```
const engagementTime = this.getEngagementTimeSeconds();
const metricData: IMetricTelemetry = {
        average: engagementTime,
        name: 'React Component Engaged Time (seconds)',
        sampleCount: 1
      };

const additionalProperties: { [key: string]: any } = { 'Component Name': componentName };
reactPlugin.trackMetric(metricData, additionalProperties);
```
I might be missing what you are seeing

Reply:
I doubt this would be a problem but just double check things are as expected.  It's a HOC which from the code I don't see a problem with but one never knows.

Reply:
I miss what I was seeing. Now that I look at it, I was mistaken.  It only captures the name like you said. Resolving this. 

But one thing this does bring up is that we are passing an entire component and we may not be aware of what type information AI captures from our component.